src/Tizen.Multimedia/MediaTool/MediaPacket.cs

```diff
@@ -201,7 +201,7 @@ public ulong Pts
         }
 
         /// <summary>
-        /// Gets or sets the DTS(Decoding Time Stamp) value of the current packet.
+        /// Gets or sets the DTS(Decoding Time Stamp) value in nanoseconds of the current packet.
```

Comment:
/// Gets or sets the DTS(Decoding Time Stamp) value of the current packet in nanoseconds.